test/ReportPortal.Shared.Tests/Execution/Metadata/MetaAttributeFixture.cs

```diff
@@ -37,5 +37,32 @@ public void ShouldCastToItemAttribute()
             ia.Key.Should().Be("a");
             ia.Value.Should().Be("b");
         }
+
+        [Fact]
+        public void TwoMetaAttributesWithTheSameKeysAndValuesShouldBeEquals()
+        {
+            var firstAttribute = new MetaAttribute("a", "b");
+            var secondAttriute = new MetaAttribute("a", "b");
+
+            firstAttribute.Should().BeEquivalentTo(secondAttriute);
```

Comment:
Please double check whether `BeEquivalentTo` method uses `IEquatable` or `Equals`. In my experience this method is smart and can compare any types of objects.

In any case let's explicitly invoke methods under testing.

Reply:
Checked it on codecov, looks good

![image](https://user-images.githubusercontent.com/22616990/173197320-9e41162c-2209-4481-b50e-2b7adb7b044f.png)